packages/create-block/README.md

```diff
@@ -99,6 +99,53 @@ $ npm run packages-update
 ```
 Updates WordPress packages to the latest version. [Learn more](/packages/scripts#packages-update).
 
+## External Templates
+Since version 0.12.0 it is possible to use external templates hosted on NPM. These templates need to contain `.mustache` files that will be used in the scaffolding and one `template.json` for the metadata. 
+
+### Availabe Variables: 
+- `namespace`
+- `slug`
+- `title`
+- `textdomain`
+- `description`
+- `category`
+- `dashicon`
+- `license`
+- `licenseURI`
+- `namespaceSnakeCase`
+- `slugSnakeCase`
+
+### `template.json`
+```json
+{
+    "defaultValues": {
+        "namespace": "create-block",
+        "slug": "esnext-example",
+        "title": "ESNext Example",
+        "description":
+            "Example block written with ESNext standard and JSX support – build step required.",
+        "dashicon": "smiley",
+        "category": "widgets",
+        "author": "The WordPress Contributors",
+        "license": "GPL-2.0-or-later",
+        "licenseURI": "https://www.gnu.org/licenses/gpl-2.0.html",
+        "version": "0.1.0"
+    },
+    "outputFiles": [
```

Comment:
For the future:

I'm very interested to find a way to automate the discovery of templates with `glob` call or something like this. It would be great to be able to default to `templates` folder that the template author can override:
```json
{
    "templatesFolder": "./my-folder"
}
```